packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewDefault.css

```diff
@@ -0,0 +1 @@
+@import './quanticQuickviewContent.css';
```

Comment:
Do you really need to make it's own css file? Does it not load the `quanticQuickviewContent.css` for both templates if you don't make those specific css files?

Reply:
I do unfortunately. LWC doesnt support the import of CSS files in js and I didn't want to add inline styles. Salesforce links the CSS files with corresponding HTML files. Its seems the best solution for now is 1 shared css file with imports in name-specific css files. Unless you know another way?

Reply:
I'm also surprised that each .html template needs its own .css 🤔 but I haven't checked so if you say it's necessary so maybe it is.